pkg/components/util/install.go

```diff
@@ -108,6 +81,7 @@ func install(helmAction *helmAction, namespace string) error {
 	install := action.NewInstall(helmAction.actionConfig)
 	install.ReleaseName = helmAction.releaseName
 	install.Namespace = namespace
+	install.CreateNamespace = true
```

Comment:
This has been added "back" (as it got removed in v3.0.0) in v3.2.0: https://github.com/helm/helm/commit/694cf61aacba6bc4ed59a65d23acd20f6c8b95bd, that's why we needed this code in the past.

Reply:
@invidian If Helm is taking care of creating namespace, then I'd wait for this merge for the PR #646 as then we avoid create step of namespace.

nit: seems this change doesn't belong in this PR.

comments like 
```
This has been added "back" (as it got removed in v3.0.0) in v3.2.0: helm/helm@694cf61, 
that's why we needed this code in the past.
```

will be lost , if you can put the same comment in a separate PR that would be great.

Reply:
True, this change does not really belong to this PR. I'll remove it. There is also an interest to get that in in #686. Also, if we let Helm to create the namespace, applying labels must happen **after** Helm release is installed. This may actually introduce some nasty bugs I imaging. Consider following scenario:

Perform Helm install action, with `Wait: true`. Installed manifests require bi-namespace NetworkPolicy objects to be created. One in component's namespace and other in component interacting with this namespace. The installation will never converge, as the namespace labels will be missing, so the network policies will not be correctly applied.

Reply:
Is this hypothesis or backed by empirical data?

Reply:
>Is this hypothesis or backed by empirical data?

I'd say neither. It's a logical conclusion. But yeah, more like hypothesis, as I didn't do tests to confirm that.